urllib3/contrib/pyopenssl.py

```diff
@@ -217,7 +217,10 @@ def shutdown(self):
 
     def close(self):
         if self._makefile_refs < 1:
-            return self.connection.close()
+            try:
+                return self.connection.close()
+            except OpenSSL.SSL.Error:
+                return
```

Comment:
Do we want to return `None` here?

The rest of this function implicitly returns `None` (e.g., when `self._makefile_refs >= 1`) so why not just do:

```py
try:
    return self.connection.close()
except OpenSSL.SSL.Error:
    # Nothing to do here
    pass
```